samples/frameworks/react-native-expo/ts/messaging/scripts/reset-project.js

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env node
```

Comment:
Did you consider writing this in TypeScript instead? Should be easy to execute with tsx.

Reply:
This files comes with the Expo template.  Only changes on top of the template project are described in README.